modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -4,37 +4,14 @@
 
 :_module-type: PROCEDURE
 [id="ztp-configuring-ptp-fast-events_{context}"]
-= Configuring PTP fast events using PolicyGenTemplate custom resources and GitOps ZTP
+= Configuring PTP fast events using PolicyGenTemplate CRs
 
 You can configure PTP fast events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline. Use `PolicyGenTemplate` custom resources (CRs) as the basis to create a hierarchy of configuration files tailored to your specific site requirements.
 
-The `PolicyGenTemplate` CRs that are relevant to PTP events can be found in the `/home/ztp/argocd/example` folder in the `quay.io/redhat_emp1/ztp-site-generator:latest` reference architecture container image. The reference architecture has a `/policygentemplates` and `/siteconfig` folder. The `/policygentemplates` folder has common, group, and site-specific configuration CRs. Each `PolicyGenTemplate` CR refers to other CRs that are in the `/source-crs` folder of the reference architecture.
-
-The `PolicyGenTemplate` CRs required to deploy PTP fast events are described below.
-
-.PolicyGenTemplate CRs for vRAN deployments
-[cols=2*, options="header"]
-|====
-|PolicyGenTemplate CR
-|Description
-
-|`common-ranGen.yaml`
-|Contains the common RAN policies that get applied to all clusters. To deploy the PTP Operator to your clusters, you configure `Namespace`, `Subscription`, and `OperatorGroup` CRs.
-
-|`group-du-3node-ranGen.yaml`
-|Contains the RAN policies for three-node clusters only, including PTP fast events configuration.
-
-|`group-du-sno-ranGen.yaml`
-|Contains the RAN policies for single-node clusters only, including PTP fast events configuration.
-
-|`group-du-standard-ranGen.yaml`
-|Contains the RAN policies for standard three control-plane clusters, including PTP fast events configuration.
-|====
-
 .Prerequisites
 
 * Create a Git repository where you manage your custom site configuration data.
-* Extract the contents of the `/home/ztp` folder from the `quay.io/redhat_emp1/ztp-site-generator:latest` reference architecture container image, and review the changes.
+* Extract the contents of the `/home/ztp` folder from the `ztp-site-generate` reference architecture container image, and review the changes.
 
 .Procedure
```

Comment:
Please change "amq-sub-policy" to "subscriptions-policy" in AMA deployment procedures.